test/mp4-emsg.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var QUnit = require('qunit'),
```

Comment:
Would it be useful to also have a test or two for invalid data?

Reply:
Good call, I'll throw together a few new tests with malformed data.